CHANGELOG.md

```diff
@@ -1,5 +1,96 @@
 ## Unreleased
 
+BREAKING CHANGES:
+
+* `consul-k8s` `v0.12.0`+ is now required. The chart is passing new flags that are only available in this version.
+  To use this version, set
```

Comment:
This is the default, right? so we don't need to tell people to set it.

Reply:
Updated the phrasing to reflect that.

Reply:
Yes it's the default but I'd guess that some folks pin it in their local values.yaml so they can upgrade the chart without also upgrading consul-k8s without doing it on purpose.